Library/Thaumatology/Thaumatology - Threshold Magic Traits.adq

```diff
@@ -0,0 +1,377 @@
+{
+	"type": "trait_list",
+	"version": 4,
+	"rows": [
+		{
+			"id": "a3d9e37d-b69d-47c2-bdaf-a565ffaf7353",
+			"type": "trait",
+			"name": "Magery",
```

Comment:
I have juggled this forth and back a few times and am not entirely satisfied either way, but ended up with "Magery (Threshold-based, +0%)" with only that modified variant affecting other Threshold ads/disads because threshold mages will generally learn all the standard spells - and those all have more or less complicated prereq chains requiring "Magery". It would be a total hassle to add a seperate "Threshold Magery" or "Magery (Threshold)" trait to all those.